core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goal-configuration-url-parameter-component/dot-experiments-goal-configuration-url-parameter-component.component.html

```diff
@@ -7,14 +7,15 @@ <h3>{{ 'experiments.goal.reach_page.form.conditions.label' | dm }}</h3>
             formArrayName="conditions"
         >
             <div class="condition flex" [formGroupName]="i">
-                <div class="field flex-grow-1 flex flex-column w-15rem">
+                <div class="field flex-grow-1 flex flex-column w-15rem" formGroupName="value">
                     <label dotFieldRequired for="value">{{
                         'experiments.goal.url_parameter.form.conditions.parameter.label' | dm
                     }}</label>
                     <input
-                        id="parameter"
+                        id="name"
                         data-testId="parameter-input"
```

Comment:
not important, but change the reference to `name-input`

Reply:
Done!